dominion/frontend/src/assets/css/index.css

```diff
@@ -0,0 +1,12 @@
+html {
```

Comment:
We should try not to use general css files like this. All components should have styles defined in the components themselves.